src/components/index.ts

```diff
@@ -74,8 +74,5 @@ export { ContentPage } from './ContentPage'
 export { PricingPlanProperty, PricingPlanFeature, PricingPlan } from './Pricing'
 export type { Features } from './Pricing'
 
-// Webinars
-export { WebinarLayout } from './WebinarLayout'
-
-// Guides
-export { GuideLayout } from './GuideLayout'
+// Webinars & Guides
+export { WebinarAndGuideLayout } from './WebinarsAndGuides'
```

Comment:
The directory name should match the component name. Also wondering if there's a better name that fits for both 💡 

```suggestion
export { WebinarAndGuideLayout } from './WebinarAndGuideLayout'
```

Reply:
Realizing now that this layout will be used for general report pages too (Issue #5369). So I'm thinking of naming the component `SimpleGatedPageLayout`, does that sound good?

Reply:
The new name for the component looks good 👍